code/game/objects/structures/morgue.dm

```diff
@@ -132,7 +132,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 	playsound(src, 'sound/effects/roll.ogg', 5, 1)
 	playsound(src.loc, 'sound/items/deconstruct.ogg', 50, 1)
 	for(var/atom/movable/AM in connected.loc)
-		if(!AM.anchored || AM == connected)
+		if(!AM.anchored || AM == connected || isliving(AM))

```

Comment:
did you test this

pretty sure ghosts aren't anchored, and they'd be valid still.

Reply:
but thats why i had THE IS MOB AND ISLIVING THING AAAAAAAAAA